ci-operator/config/openshift-power-monitoring/kepler/openshift-power-monitoring-kepler-main.yaml

```diff
@@ -41,8 +41,11 @@ tests:
       cli: latest
       commands: |
         GOFLAGS="" make tools
-        make build-manifest OPTS="OPENSHIFT_DEPLOY"
+         GOFLAGS="" make build-manifest OPTS="OPENSHIFT_DEPLOY"
         oc apply -f _output/generated-manifest/deployment.yaml
+      dependencies:
+      - env: EXPORTER_IMG
+        name: kepler
```

Comment:
How is this being used? ... may be in a another PR, we should use a more precise name than exporter :)